Uplift/Controllers/FilterViewController.swift

```diff
@@ -143,14 +147,22 @@ class FilterViewController: UIViewController, RangeSeekSliderDelegate {
         contentView.addSubview(startTimeClassTypeDivider)
 
         // CLASS TYPE SECTION
-        classTypeDropdown = UITableView(frame: .zero, style: .grouped)
+        classTypeDropdownHeader = DropdownHeaderView(frame: .zero)
+        contentView.addSubview(classTypeDropdownHeader)
+        
+        classTypeDropdownFooter = DropdownFooterView(frame: .zero)
+        classTypeDropdownFooter.clipsToBounds = true
+        contentView.addSubview(classTypeDropdownFooter)
+        
+        let classTypeGesture = UITapGestureRecognizer(target: self, action: #selector(self.dropHideClasses(sender:) ))
+        classTypeDropdownFooter.addGestureRecognizer(classTypeGesture)
+        
+        classTypeDropdown = UITableView(frame: .zero)
         classTypeDropdown.separatorStyle = .none
         classTypeDropdown.showsVerticalScrollIndicator = false
         classTypeDropdown.bounces = false
-
+        
```

Comment:
why are there new lines here

Reply:
sometimes I click "enter" and they just happen.